# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../mine/cdb-gcc6.patch
# Copyright (C) 2018 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- mine-0.23/cdb-0.75/cdbmake.c.vanilla	2017-12-29 23:33:39.854936413 +0000
+++ mine-0.23/cdb-0.75/cdbmake.c	2017-12-29 23:34:05.226935484 +0000
@@ -26,7 +26,7 @@
   strerr_die2x(111,FATAL,"unable to read input: bad format");
 }
 
-inline void get(char *ch)
+static inline void get(char *ch)
 {
   switch(buffer_GETC(buffer_0,ch)) {
     case 0: die_readformat();
